src/main/java/uk/gov/justice/hmpps/prison/api/resource/CaseNoteResource.java

```diff
@@ -101,7 +113,10 @@ public List<CaseNoteTypeCount> getCaseNoteUsageSummaryByDates(@RequestBody @Para
             @ApiResponse(responseCode = "400", description = "Invalid request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
             @ApiResponse(responseCode = "404", description = "Requested resource not found.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
             @ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})})
-    @Operation(summary = "Count of case notes by booking id", description = "Count of case notes by booking id. Requires role VIEW_CASE_NOTES")
+    @Operation(
+        deprecated = true,
+        summary = "This endpoint has been replaced in the case notes service - see case-notes-service/case-notes/usage"
```

Comment:
this is the `usage-by-types` endpoint - is there anything similar in the case notes service?